### PR TITLE
Multi tenant Examples

### PR DESCRIPTION
**Pull request by pboling**
I got it working!

At least the parts that are tested.  I'll add more.

I based this on top of branch https://github.com/janko/rodauth-rails/tree/url-helpers (PR #245) but also rebased on top of latest main as of [71eaf3d](https://github.com/janko/rodauth-rails/commit/71eaf3d87bb3068ac4d484c92f8ee51a97e9c646).

I did this because it looked like the `url-helpers` branch was building in the direction of what I needed.

### CODE REVIEW

**Comment by janko** [comment]
rodauth-rails doesn't have any special multi-tenancy support, so it wouldn't really make sense to add these examples to the test suite.

What would IMO be far more useful is a wiki page with instructions on how to configure rodauth-rails for multi-tenancy 😉 The wiki is publicly editable, so feel free to add a new page.

File: test/rails_app/app/misc/rodauth_multi_tenant.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      check_csrf? { rails_request.format.turbo_stream? ? false : super() }
+    end
+
+    after_login { remember_login }
+
+    logout_redirect { rails_routes.root_path }
+    login_redirect do
+      segs = login_path.split('/')
+      segs.insert(-2, request.env[:path_key])
+      segs.join('/')
```
Comment (janko) [inline test/rails_app/app/misc/rodauth_multi_tenant.rb:53]: Why is this needed, BTW? Won't the overridden `#prefix` method automatically add the `path_key`.

It seems you have two sources of `path_key` – the attribute accessor and `request.env`. Maybe that's why it isn't working?

Comment (pboling) [reply test/rails_app/app/misc/rodauth_multi_tenant.rb:53]: That shouldn't be there... It was one of my transitional hacks as I figured it out... Not sure why it is still there.

Comment (pboling) [reply test/rails_app/app/misc/rodauth_multi_tenant.rb:53]: I'll add a wiki page once I have it all setup and working!

**Comment by whysthatso** [comment]
@pboling have you ever had a chance to write down your findings? i am considering using [acts_as_tennant](https://github.com/ErwinM/acts_as_tenant), and i'm wondering if i understand it enough to just go for it and try it out, or if this needs more work.
